src/App.js

```diff
@@ -1,15 +1,18 @@
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Home from './components/page/Home';
-import ProductReg from './components/page/ProductReg';
-import ProductList from './components/page/ProductList';
+import ProductListPage from './components/page/ProductListPage';
+import ProductRegPage from './components/page/ProductRegPage';
+import ProductDetailPage from './components/page/ProductDetailPage';
+
 
 function App() {
   return (
     <BrowserRouter>
       <Routes>
         <Route path='/' element={<Home />} />
-        <Route path='/items' element={<ProductList />} />
-        <Route path='/additem' element={<ProductReg />} />
+        <Route path='/items' element={<ProductListPage />} />
+        <Route path='/additem' element={<ProductRegPage />} />
+        <Route path='/items/:productId' element={<ProductDetailPage />} />
```

Comment:
개인적인 선호도 차이이긴 한데, `/endpoint` 들이 같은 애들끼리 묶어두면 좀 더 관리할때 수월하답니다! 
```suggestion
        <Route path='/items' element={<ProductListPage />} />
        <Route path='/items/:productId' element={<ProductDetailPage />} />
        <Route path='/additem' element={<ProductRegPage />} />
```